Resources/Private/Layouts/Default.html

```diff
@@ -7,7 +7,19 @@
 		<div id="typo3-docheader">
 			<div class="typo3-docheader-functions">
 				<div class="left">
-					<solr:backend.menu.SiteSelectorMenu name="site" />
+
+					<solr:backend.sites>
+						<f:if condition="{hasSites}">
+							<f:then>
+								<solr:backend.menu.SiteSelectorMenu name="site" />
+							</f:then>
+							<f:else>
+								No solr connections initialized!
```

Comment:
In general I think I have to check the empty state again - what is shown the user hits the module the first time. Would be nice to offer a little message and a button to initialize the sites.